CombinatorialGames/Surreal/Division.lean

```diff
@@ -301,3 +320,33 @@ theorem mk_ratCast (q : ℚ) : mk q = q := by
   · exact_mod_cast q.den_pos
 
 end Surreal
+
+namespace IGame
+
+-- TODO: upstream
+attribute [simp] AntisymmRel.refl
+
+private theorem equiv_ratCast_of_mem_move_inv_natCast {n : ℕ} :
+    (∀ x ∈ leftMoves.{u} (n + 1)⁻¹, ∃ q : ℚ, x ≈ q) ∧
+      (∀ x ∈ rightMoves.{u} (n + 1)⁻¹, ∃ q : ℚ, x ≈ q) := by
+  refine invRec _ ⟨0, ?_⟩ ?_ ?_ ?_ ?_
+  any_goals simp
```

Comment:
This solves three of the five goals, and is a non-terminal simp in the other two, which happens to simplify favorably. I'm not aware of any way to write down "`simp`, and if it doesn't solve the goal, fail", but this should probably be replaced by that.

Reply:
You could use `try · tac`.